sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs

```diff
@@ -144,7 +144,11 @@ public void Upsert_Simple_Order()
                             { $"Order/{testScope}/Account", new PerpetualProperty($"Order/{testScope}/Account", new PropertyValue("J Wilson")) },
                             { $"Order/{testScope}/Strategy", new PerpetualProperty($"Order/{testScope}/Strategy", new PropertyValue("RiskArb")) },
                         },
-                        side: "Buy"
+                        
+                        side: "Buy", 
+                        state: "New", 
+                        type: "Limit", 
+                        date: DateTimeOffset.Parse("2022-07-02")
```

Comment:
are/should these be cut labels?

Reply:
I don't know what a cut label is

Reply:
@timbarrass-finbourne should these be cut labels?

Reply:
Cut labels aren't supported for order entity date fields at present